src/main/java/frc/robot/RobotContainer.java

```diff
@@ -57,14 +52,12 @@ private void configureButtonBindings() {
     driverStick = new Joystick(0);
     controlStick = new Joystick(1);
 
-    // Instantiate the drive command and bind it
-    driveCommand = new ArcadeDriveWithJoystick(driverStick, Config.LEFT_CONTROL_STICK_Y, Config.INVERT_FORWARD, Config.RIGHT_CONTROL_STICK_X, Config.INVERT_SIDE);
+    /**
+     * Select drive mode for robot
+     */
 
+    driveCommand = new ArcadeDriveWithJoystick(driverStick, Config.LEFT_CONTROL_STICK_Y, Config.INVERT_FIRST_AXIS, Config.RIGHT_CONTROL_STICK_X, Config.INVERT_SECOND_AXIS);
     DriveBase.getInstance().setDefaultCommand(driveCommand);
-
-    // Instantiate the intake command and bind it
-    intakeCommand = new OperatorIntakeCommand();
```

Comment:
Is there a reason these 2 lines are being deleted?

Reply:
No I think that was an accident I will re add